portal/eproms_substudy_tailored_content/src/js/components/App.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <div id="app">
+        <div id="loadingIndicator" class="loader" v-show="isLoading()"></div>
+        <header-section></header-section>
+        <main>
+            <intro-section v-bind:content="introContent"></intro-section>
+            <!--empty and content populated dynamically-->
+            <domain-section v-bind:content="domainContent"></domain-section>
+            <!-- TODO will there be a resource section? if so have a separate template section -->
+            <error></error>
+        </main>
+    </div>
+</template>
+<script>
+    import "custom-event-polyfill";
+    import bootstrap from "bootstrap";
+    import $ from 'expose-loader?exposes[]=$&exposes[]=jQuery!jquery';
+    import "bootstrap/dist/css/bootstrap.min.css";
+    import "../../style/app.less";
```

Comment:
We may have to copy some code from the portal repo. Do you expect that to be much? If so, we may want to think about how to share components common to these without copy/paste.

Reply:
ivan, so I have tried to avoid having to copy any code from the portal repo except perhaps the css.  I don't expect to do that much for this particular app.  But it will be good to think about sharing components down the road.